src/main/resources/reference.conf

```diff
@@ -54,7 +54,10 @@ sca-wrapper {
       pertax-frontend.url = "http://localhost:9232"
       accessibility-statement-frontend.url = "http://localhost:12346"
       single-customer-account-wrapper-data.url = "http://localhost:8422"
+      help-improve-banner.url = "https://signup.take-part-in-research.service.gov.uk/home?utm_campaign=PTAhomepage&utm_source=Other&utm_medium=other&t=HMRC&id=209"
```

Comment:
utm_campaign should probably be a parameter that a service can set in their application.conf file